src/components/Nav/Nav.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import { GiAtSea, GiHamburgerMenu } from 'react-icons/gi';
+import { GoSearch } from 'react-icons/go';
+import { BsHandbag, BsInstagram, BsYoutube } from 'react-icons/bs';
+import { GrClose } from 'react-icons/gr';
+import { useState } from 'react';
+import './Nav.scss';
+
+const SearchBox = ({ handleSearchClick }) => {
+  return (
+    <>
+      <GrClose onClick={handleSearchClick} className="searchClose" />
+      <section className="searchBox">
+        <input type="text" placeholder="SEARCH" />
+        <span>검색어 입력후에 엔터를 누르세요.</span>
+      </section>
+    </>
+  );
+};
+
+const Logo = () => {
+  return (
+    <ul className="logo">
+      <GiAtSea className="logoIcon" />
+      <div className="logoText">
+        <span>CORWSEVEN</span>
+        <span>HOME</span>
+        <span>SEOUL KOREA</span>
+      </div>
+    </ul>
+  );
+};
+
+const RightBoxNav = ({ isToggle, handleClick }) => {
+  return (
+    <div className={isToggle ? 'rightNavBox widthZero' : 'rightNavBox'}>
```

Comment:
```suggestion
    <div className={`rightNavBox ${isToggle ? widthZero : ""}`}>
```

항상 들어가는 rightNavBox className도 삼항을 이용해 표현할 필요는 없겠네요!